minseo/makeModules.js

```diff
@@ -0,0 +1,117 @@
+require("dotenv").config();
+const { DataSource } = require("typeorm");
+const appDataSource = new DataSource({
+  type: process.env.DB_CONNECTION,
+  host: process.env.DB_HOST,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+  port: process.env.DB_PORT,
+});
+
+appDataSource
+  .initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  })
+  .catch((err) => {
+    console.log("Error during Data source initialization", err);
+    appDataSource.destroy();
+  });
+
+const update_post = async (req, res) => {
```

Comment:
자바스크립에서는 카멜케이스를 기본 컨벤션으로 가져가고 있습니다. 스네이크 케이스로 작성된 함수명을 전부 카멜 케이스로 변경해 주세요.
```suggestion
const updatePost = async (req, res) => {
```